frontend/src/pages/signup.jsx

```diff
@@ -1,90 +1,109 @@
+      });
+  };
 
   return (
-    <main >        
-        <section>
-            <div>
-                <div>                  
-                    <form>                                                                                            
-                        <div>
-                            <label htmlFor="email-address">
-                                Email address
-                            </label>
-                            <input
-                                type="email"
-                                label="Email address"
-                                value={email}
-                                onChange={(e) => setEmail(e.target.value)}  
-                                required                                    
-                                placeholder="Email address"                                
-                            />
-                        </div>
-
-                        <div>
-                            <label htmlFor="password">
-                                Password
-                            </label>
-                            <input
-                                type="password"
-                                label="Create password"
-                                value={password}
-                                onChange={(e) => setPassword(e.target.value)} 
-                                required                                 
-                                placeholder="Password"              
-                            />
-                        </div>                                             
-
-                        <button
-                            type="submit" 
-                            onClick={onSubmit}                        
-                        >  
-                            Sign up                                
-                        </button>
+    <main className="bg-customCream h-screen flex items-center justify-center">
+      {/* Logo in the top left corner with link back to homepage */}
+      <Link
+        to="/"
+        className="text-customBlue text-4xl font-bold font-inknut absolute top-4 left-6"
+      >
+        SKINrecs
+      </Link>
+      <div className="w-full max-w-md">
+        {/* Sign up prompt */}
+        <div className="bg-customBlue shadow-lg rounded-lg p-8">
+          <h2 className="text-center text-customCream text-3xl font-bold mb-6 font-inknut">
+            Sign Up
+          </h2>
+          <form>
+            {/* Input email address */}
+            <div className="mb-4">
+              <label htmlFor="email-address" className="sr-only">
+                Email address
+              </label>
+              <input
+                id="email-address"
+                name="email"
+                type="email"
+                label="Email address"
+                value={email}
+                required
+                className="w-full p-3 rounded border border-gray-300"
```

Comment:
consider text color customGray

Reply:
just changed the text to fit the search bar